src/common/components/serviceNotifications/ServiceNotifications.tsx

```diff
@@ -7,8 +7,13 @@ import { useTranslation } from 'react-i18next';
  * in Haitaton service
  */
 function ServiceNotifications() {
+  console.log(useTranslation());
```

Comment:
There seems to be `console.log` left here.

Reply:
Thank you for noticing! I removed it.